src/fromager/wheels.py

```diff
@@ -111,7 +111,7 @@ def build_wheel(
     wheels = list(ctx.wheels_build.glob("*.whl"))
     if wheels:
         return wheels[0]
-    return None
+    raise FileNotFoundError(f"Could not locate the built wheels for {req.name}")
```

Comment:
The signature of the exception class is a bit obscure. It takes an errno number, error string, and file or target name:

```suggestion
    raise FileNotFoundError(errno.ENOENT, "Could not locate the built wheels for", req.name)
```

```python
>>> import errno
>>> from packaging.requirements import Requirement
>>> req = Requirement("egg==0.1")
>>> raise FileNotFoundError(errno.ENOENT, "Could not locate the built wheels for", req.name)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] Could not locate the built wheels for: 'egg'
```